parse.js

```diff
@@ -0,0 +1,26 @@
+export default function(data){
+
+    var nrows = data.length;
+
+    if (nrows === undefined || nrows === null){
+        keys = ['message'];
+        nrows = 1;
+        ncols = 1;
+        rows = [['command executed']];
+    }else{
```

Comment:
let's get eslint going on our machines on monday so that we can have super consistent style